iotdb/src/main/java/org/apache/iotdb/db/writelog/transfer/SystemLogOperator.java

```diff
@@ -28,8 +28,8 @@ private SystemLogOperator(){}
   public static final int INSERT = 0;
   public static final int UPDATE = 1;
   public static final int DELETE = 2;
-  public static final int OVERFLOWFLUSHSTART = 3;
-  public static final int OVERFLOWFLUSHEND = 4;
-  public static final int BUFFERFLUSHSTART = 5;
-  public static final int BUFFERFLUSHEND = 6;
```

Comment:
overflow and buffer is not used?

Reply:
Yes, these are all unused, so I remove them.